index.js

```diff
@@ -93,20 +92,21 @@ module.exports = {
     // and an intended value `val` (which may be an object containing any of the
     // documented operators for this module such as ``$append`), return the
     // new value that is appropriate for `path`.
-    
+
     self.getNewOptionValue = function(req, moduleOptions, sliced, val) {
-      if (val && (typeof(val) === 'object')) {
+      if (val && (typeof (val) === 'object')) {
```

Comment:
Not sure if you need the parens around the val. Don't think it's hurting anything but don't think they're necessary.